convertCSVtoJSON.js

```diff
@@ -3,6 +3,7 @@ const csv = require("csvtojson");
 
```

Comment:
This logic might be clearer if you used a switch statement instead of multiple if-else statements.

Reply:
he kay mhantoys kai kalat nahi re baba